R/github.R

```diff
@@ -79,20 +79,33 @@ github_pat <- function(quiet = TRUE) {
     pat <- Sys.getenv(env_var)
     if (nzchar(pat)) {
       if (!quiet) {
-        message("Using github PAT from envvar ", env_var)
+        message("Using github PAT from envvar ", env_var, ". ",
+                "Use `gitcreds::gitcreds_set()` if you want to use the git ",
+                "credential store instead")
```

Comment:
Elsewhere, I've also tried to nag people to proactively undo whatever was setting `GITHUB_PAT` or `GITHUB_TOKEN` in the first place, e.g. to open `.Renviron` and delete the relevant line. It feels like you should probably do same, if you're going to open this can of worms.

Reply:
Yeah. I figured (some) people probably have a good reason to use the env var. (e.g. cron job on Linux, or just Linux in general, where the credential helpers are really lacking), so I did not want to annoy them too much with a longer message. But I'll make it a bit longer to note that they can delete the env var.